addons/network/XEH_preInit.sqf

```diff
@@ -95,8 +93,38 @@ if (SLX_XEH_MACHINE select 3) then {
     };
 };
 
+// CBA_fnc_globalExecute is normally enabled
+private _disableGE = false;
+
+// Check the CfgSettings >> "CBA" >> "network" >> "disableGlobalExecute" value
+private _entry = (CFGSETTINGSS(CBA,COMPONENT) >> "disableGlobalExecute");
+if (isNumber _entry) then {
+    _disableGE = (getNumber _entry == 1);
+} else {
+    if (isText _entry) then {
+        _disableGE = (getText _entry == "true");
+    };
+};
+
+// Check the mission configuration
+private _dGE = getMissionConfigValue ["CBA_disableGlobalExecute", -1];
+if (_dGE isEqualType 0) then {
+    if (_dGE in [0, 1]) then {
+        _disableGE = (_dGE == 1);
+    };
+} else {
+    if (_dGE isEqualType "") then {
+        _disableGE = (_dGE == "true");
+    };
+};
 
-[QUOTE(GVAR(cmd)), { if (GVAR(init)) then { _this spawn FUNC(exec) } }] call (uiNamespace getVariable "CBA_fnc_addEventHandler");
+if (_disableGE) then {
+    uiNamespace setVariable [QGVAR(exec), compileFinal ""];

```

Comment:
Should this be `QUOTE(DFUNC(exec))`?

I don't think we want to use uiNameSpace / caching for this function at all.
Otherwise I can create a mission with it disabled, and it will compileFinal the function to `{}` in UI
then join a server with it enabled and PREP will still pull the `{}`.